src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -21,6 +21,8 @@ import scala.util.{Success, Try, Failure}
  * Created by dvoet on 10/27/15.
  */
 object UserService {
+  val allUsersGroupRef = RawlsGroupRef(RawlsGroupName("All_Users"))
```

Comment:
only admins can create or update groups and they should not do naughty things but they have full control